src/Microsoft.VisualStudio.Editors/PropPages/BuildPropPage.resx

```diff
@@ -1449,4 +1449,7 @@
   <data name="&gt;&gt;$this.Type" xml:space="preserve">
     <value>Microsoft.VisualStudio.Editors.PropertyPages.PropPageUserControlBase, Microsoft.VisualStudio.AppDesigner, Version=42.42.42.42, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="cboWarningLevel.Items5" xml:space="preserve">
+    <value>5</value>
+  </data>
```

Comment:
Is there a way to mark a specific resource string as non-localisable?

Reply:
Possibly but I couldn't figure it out. @tmeschter Do you know? I'll file a followup to fix if there is, this just following the existing pattern.

Reply:
The best option is to just keep the value in code rather than in a .resx file.

Second best option is to add a `<comment>` element (or maybe it is `<note>`; I forget) to the `<data>` item in the .resx file and state that the text should be left as-is across all languages. Those notes are copied to the .xlf files and are visible to the translators.